Formula/musikcube.rb

```diff
@@ -0,0 +1,49 @@
+require "hardware"
+
+class Musikcube < Formula
+  desc "Terminal-based audio engine, library, player and server"
+  homepage "https://musikcube.com"
+  url "https://github.com/clangen/musikcube/archive/0.99.2.tar.gz"
+  sha256 "782ee8aba01b5e5cf4f5959570d087d7a1208a9309e5ae16074894801a8544bf"
+  license "BSD-3-Clause"
+  head "https://github.com/clangen/musikcube.git", branch: "master"
+
+  depends_on "ccache" => :build
+  depends_on "cmake" => :build
+  depends_on "curl"
+  depends_on "ffmpeg"
+  depends_on "game-music-emu"
+  depends_on "gnutls"
+  depends_on "lame"
+  depends_on "libev"
+  depends_on "libmicrohttpd"
+  depends_on "libogg"
+  depends_on "libopenmpt"
+  depends_on "libvorbis"
+  depends_on "ncurses"
+  depends_on "openssl@1.1"
+  depends_on "taglib"
+
+  def install
+    @job_count = [Hardware::CPU.cores - 2, 1].max
+    system "git",
+              "clone",
+              "--branch",
+              "asio-1-24-0",
+              "https://github.com/chriskohlhoff/asio/",
+              "src/3rdparty/asio/"
```

Comment:
This should use the brewed version

Reply:
https://github.com/Homebrew/homebrew-core/pull/118938/commits/785f116167f4b1f242bbba336362f6a5d2c97e97#diff-ca6572917efe0a188596908555ec5c292dacb9eefc0f2969ca50f6a6b9e31a26R11